torchvision/models/regnet.py

```diff
@@ -587,6 +587,21 @@ class RegNet_Y_16GF_Weights(WeightsEnum):
             "acc@5": 98.054,
         },
     )
+    IMAGENET1K_SWAG_LINEAR_V1 = Weights(
+        # This model has trunk weight from weakly supervised learning described in https://arxiv.org/pdf/2201.08371.pdf
+        # with only linear head finetuned to IMAGENET1K dataset
+        # This model is suitable for users that want to fine tune the pretrained trunk on other downstream datasets
+        url="https://download.pytorch.org/models/regnet_y_16gf_lc_swag-f3ec0043.pth",
+        transforms=partial(
+            ImageClassification, crop_size=224, resize_size=224, interpolation=InterpolationMode.BICUBIC
+        ),
+        meta={
+            **_COMMON_SWAG_META,
```

Comment:
I would change the recipe URL from the SWAG repo to this PR for these models. On the description of this PR you can add information such as links to the original repo/paper, how exactly this was trained, the hyperparams used to train the linear layer provided by @lauragustafson / @mannatsingh etc. This is only temporary because we plan to create more proper documentation for the recipes.

Reply:
@datumbox Yeah, I intend to do that. I have updated the recipe and add the training params on the PR description as well. Please help me review the PR description as well and let me know if you think I miss anything. Thanks!